src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Safeguarding.cshtml

```diff
@@ -1,8 +1,12 @@
 ﻿@page
 @model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.SafeguardingModel
+@{
+    ViewData["Title"] = "Safeguarding";
+    string serviceName = System.Web.HttpUtility.UrlEncode(Model.ServiceName);
+}
 
 @section Back {
-    <a asp-page="/ProfessionalReferral/LocalOfferDetail" asp-route-serviceId="@Model.Id" asp-route-name="@Model.Name" class="govuk-back-link">Back</a>
+    <a href="/ProfessionalReferral/LocalOfferDetail/c2e8cebb-4597-4032-89c7-2559cdce33e8/?serviceId=@Model.ServiceId" class="govuk-back-link">Back</a>
```

Comment:
The dummy id has been removed, so I don't think we need the GUID.

Reply:
Does not work without it

Reply:
Try updating from the base branch.